provider/resource_keycloak_openid_client_test.go

```diff
@@ -753,20 +753,18 @@ func TestAccKeycloakOpenidClient_loginTheme(t *testing.T) {
 }
 
 func TestAccKeycloakOpenidClient_import(t *testing.T) {
-	t.Parallel()
-
 	resource.Test(t, resource.TestCase{
 		ProviderFactories: testAccProviderFactories,
 		PreCheck:          func() { testAccPreCheck(t) },
 		CheckDestroy:      testAccCheckKeycloakOpenidClientNotDestroyed(),
 		Steps: []resource.TestStep{
 			{
 				Config:      testKeycloakOpenidClient_import("non-existing-client", true),
-				ExpectError: regexp.MustCompile("Error: openid client with name non-existing-client does not exist"),
```

Comment:
previous error message was not consistent with KeyCloak glossary: it must be "Client ID", not "name"